src/m365/aad/commands/m365group/m365group-recyclebinitem-remove.ts

```diff
@@ -150,7 +150,8 @@ class AadM365GroupRecycleBinItemRemoveCommand extends GraphCommand {
     }
 
     if (groups.length > 1) {
-      throw Error(`Multiple groups with name '${displayName || mailNickname}' found: ${groups.map(x => x.id).join(',')}.`);
+      const resultAsKeyValuePair = formatting.convertArrayToHashTable('id', groups);
+      groups[0] = await Cli.handleMultipleResultsFound<Group>(`Multiple groups with name '${displayName || mailNickname}' found.`, resultAsKeyValuePair);
```

Comment:
having a quick check this changes is possible but we expect to return a single string, not a `Group` (at least as of now). 
So we either 
- have to refactor the command to return the `Group` or 
- wrap `handleMultipleResultsFound` around `()` and get the `.id!` 🤔 like
`return (await Cli.handleMultipleResultsFound<Group>(`Multiple groups with name '${displayName || mailNickname}' found.`, resultAsKeyValuePair)).id!;`

I'm asking since you used the code change did you try to actually do the refactor locally to check your idea? The question is which path you want to take: refactor to return object to the parent or id?
With this comment is hard to get your intention 😅

Reply:
So here: don't refactor the function. I made a mistake in my suggestion. Just return the id. my intention was to focus on returning a property in the same line as the await.